runners/google-cloud-dataflow-java/worker/windmill/src/main/proto/windmill.proto

```diff
@@ -758,7 +758,8 @@ message WorkerMetadataResponse {
   // CommitWorkStream. Each response on this stream replaces the previous, and
   // connections to endpoints that are no longer present should be closed.
   message Endpoint {
-    optional string endpoint = 1;
+    optional string direct_endpoint = 1;
+    optional string worker_token = 2;
```

Comment:
I think we want to remove direct_path_endpoints and just use work_endpoints

Reply:
done.